### PR TITLE
Hackweek: Adding mozilla www metrics and summeries to Looker

### PR DESCRIPTION
**Pull request by kik-kik**
# Hackweek: Adding mozilla www metrics and summeries to Looker

Part of Looker hackweek: https://docs.google.com/spreadsheets/d/1wFNaN2KVo_0zvhE7CwW6DLJKSgTUBoiA8ekWzOU-5y0

### CODE REVIEW

File: custom-namespaces.yaml [hunk 1]
```diff
@@ -714,3 +714,26 @@ shared:
       type: table_view
       tables:
         - table: mozdata.static.data_incidents_v1
+site:
+  glean_app: false
+  pretty_name: Site
```
Comment (gkabbz) [inline custom-namespaces.yaml:719]: Lets change pretty name to `Websites`

File: custom-namespaces.yaml [hunk 2]
```diff
@@ -714,3 +714,26 @@ shared:
       type: table_view
       tables:
         - table: mozdata.static.data_incidents_v1
+site:
```
Comment (gkabbz) [inline custom-namespaces.yaml:717]: Change name to websites

File: custom-namespaces.yaml [hunk 3]
```diff
@@ -714,3 +714,26 @@ shared:
...
+  views:
+    www_site_page_metrics:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_page_metrics_v1
+    www_site_metrics_summary:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_metrics_summary_v1
+    www_site_landing_page_metrics:
```
Comment (gkabbz) [inline custom-namespaces.yaml:732]: Lets change name to `moz_org_landing_page_metrics`

File: custom-namespaces.yaml [hunk 4]
```diff
@@ -714,3 +714,26 @@ shared:
...
+  pretty_name: Site
+  spoke: looker-spoke-default
+  owners:
+    - kignasiak@mozilla.com
+  views:
+    www_site_page_metrics:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_page_metrics_v1
+    www_site_metrics_summary:
```
Comment (gkabbz) [inline custom-namespaces.yaml:728]: Lets change name to `moz_org_metrics_summary`

File: custom-namespaces.yaml [hunk 5]
```diff
@@ -714,3 +714,26 @@ shared:
...
       tables:
         - table: mozdata.static.data_incidents_v1
+site:
+  glean_app: false
+  pretty_name: Site
+  spoke: looker-spoke-default
+  owners:
+    - kignasiak@mozilla.com
+  views:
+    www_site_page_metrics:
```
Comment (gkabbz) [inline custom-namespaces.yaml:724]: Lets change name to moz_org_page_metrics

**Review by gkabbz — CHANGES_REQUESTED** [review]

File: custom-namespaces.yaml (hunk 1 above)
Comment (kik-kik) [reply custom-namespaces.yaml]: Changes it to `Mozilla Org Websites` to make it explicit that this namespace related to Mozilla's websites

File: custom-namespaces.yaml [hunk 6]
```diff
@@ -714,3 +714,26 @@ shared:
       type: table_view
       tables:
         - table: mozdata.static.data_incidents_v1
+moz_org_sites:
```
Comment (gkabbz) [inline custom-namespaces.yaml:717]: Make this `websites`. Sometimes we have sites that are not exactly mozilla.org. Plus when marketers talk about moz org they are specifically referring to mozilla.org. e.g. Addons = addons.mozilla.org which isn't technically referred to when people thing moz org.

File: custom-namespaces.yaml [hunk 7]
```diff
@@ -714,3 +714,26 @@ shared:
...
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_page_metrics_v1
+    moz_org_site_metrics_summary:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_metrics_summary_v1
+    moz_org_site_landing_page_metrics:
+      type: table_view
+      tables:
+        - table: moz-fx-data-marketing-prod.ga_derived.www_site_landing_page_metrics_v1
+    moz_org_blogs_daily_summary:
```
Comment (gkabbz) [inline custom-namespaces.yaml:736]: Leave this as `blogs_daily_summary` blogs = blogs.mozilla.org and how people reference it internally so should have different naming than moz org = mozilla.org

**Review by gkabbz — CHANGES_REQUESTED** [review]

**Review by gkabbz — APPROVED** [review]

**Pull request merged**